### PR TITLE
fix: ask user to check wallet

### PR DESCRIPTION
**Pull request by lucachaco**
image

### CODE REVIEW

File: src/Core.tsx [hunk 1]
```diff
@@ -309,7 +309,11 @@ export class Core extends React.Component<IModalProps, IModalState> {
      const { providerController } = this.props
 
      const providerName = provider.name || 'Provider'
-
+     this.setState({
+       currentStep: 'loading',
+       loadingReason: i18next.t('Please open your wallet'),
+       selectedProviderUserOption: { ...selectedProviderUserOption, provider }
```
Comment (jessgusclark) [inline src/Core.tsx:315]: Why is `selectedProviderUserOption` set here? It looks like it should only be set once the provider has connected in the `then` right below.

**Review by jessgusclark — CHANGES_REQUESTED** [review]
Functionality works great. Quick question about a variable being set, and then can you add a Cypress test for this? Perhaps add a setTimeout on the mock web3 provider to add the time? [Something similar to this](https://github.com/rsksmart/rLogin/blob/develop/cypress/integration/1-modal/rlogin-retry-provider_spec.js#L15) but with a timeout instead to allow the wheel to spin?

**Comment by ilanolkies** [comment]
I added this PR to `mock-web3-provider` to allow us to do integration test of the loading screen https://github.com/rsksmart/mock-web3-provider/pull/15

**Comment by ilanolkies** [comment]
Now we can do something like

```ts
    cy.visit(`/`)
    cy.get('#login').click()
    cy.contains('MetaMask').click()
    cy.get('#loading').should('have.text', 'Please confirm in your wallet')

    provider.answerEnable(true)
    confirmInformationStep()
    // ... and so on
```

We can also see test what happens if `provider.answerEnable(false)`

File: src/Core.tsx (hunk 1 above)
Comment (lucachaco) [reply src/Core.tsx]: Removed

**Comment by lucachaco** [comment]
Good idea of that `mock-web3-provider` PR

**Comment by lucachaco** [comment]
Added test but https://github.com/rsksmart/mock-web3-provider still published with old version

**Pull request closed without merging**